ember.md

```diff
@@ -23,49 +23,288 @@
   [ [Versioning](#versioning) ]
 
 <a name="general-structure"></a>
+
 ## General Structure
 
 **Why do we structure our Javascript code?**
 This allows us to follow a consistent coding conventions. Paired with a tool to enforce code structure, such as Eslint, it will provide a way for us to make sure that our code follow a specific specified order.
 
 <a name="general-structure--properties"></a>
-- 1.1 **Controller/Component Property Order**: follow this order for properties and methods
-  + **Properties**
-    + Put ember specific properties (e.g., `classNames`, `tagNames`) before custom properties (e.g., `someSpecificAshProp : true`)
-  + Component lifecycle hooks (in order) <a href="https://guides.emberjs.com/v2.12.0/components/the-component-lifecycle/">see order in documentation</a>
-  + Custom methods
-  + `actions` go last
-  
-- 1.2 **Model Property Order**: follow this order for defining attributes and relationships:
-	+ Attributes
-	+ Relationships
-	+ Single line function
-	+ Multi line function
+
+### 1.1 Model Property Order
+
+1. Attributes
+1. Relations
+1. Single line computed properties
+1. Multi line computed properties
+1. Other structures (custom methods etc.)
 
 ```javascript
-//Bad
-firstName: attr('string'),
-fullName: belongsTo('person'),
-lastName: attr('string')
+import DS from 'ember-data';
+const { Model, attr, hasMany } = DS;
+import { computed, get } from '@ember/object';
 
-//Good
-firstName: attr('string'),
-lastName: attr('string'),
+// bad
+export default Model.extend({
+  mood: computed('health', 'hunger', function() {
+    const result = get(this, 'health') * get(this, 'hunger');
+    return result;
+  }),
+
+  hat: attr('string'),
+
+  behaviors: hasMany('behaviour'),
+
+  shape: attr('string')
+});
+
+// good
+export default Model.extend({
+  // 1. Attributes
+  shape: attr('string'),
+
+  // 2. Relations
+  behaviors: hasMany('behaviour'),
+
+  // 3. Computed Properties
+  mood: computed('health', 'hunger', function() {
+    const result = get(this, 'health') * get(this, 'hunger');
+    return result;
+  })
+});
+```
+Enforce by adding this rule to `.eslintrc.js`:
+```
+rules: {
```

Comment:
Really helpful mentioning how to enable the rules. I do think we should roll them up and just list way to enable them all at the top - seems unlikely that somebody would be only turning on some of these?